src/index.js

```diff
@@ -141,6 +141,11 @@ const sendNodeAPIRequest = (path, method, headers, body) => {
         _send()
       })
     } else {
+      if (method === 'DELETE' && body) {
+        const Buffer = require('buffer').Buffer
+        options.headers['content-length'] = Buffer.byteLength(body)
+      }
```

Comment:
Please remove the check for `DELETE` method - we don't need it
Instead make sure that the `content-length` header wasn't set explicitly by user. Do nothing in that case